### ISSUE TITLE
Kraken good accessibility - Completed

### ISSUE THREAD

**Issue by KarenMendozaGarza** [KarenMendozaGarza/kraken-good-accessibility#2]
Hi!

I have completed the activity. Please let me know what changes I may need to make:)

Have a great week,

Karen Mendoza
030

@thatkidmadrid

**Comment by thatkidmadrid** [comment]
Just jumped in and first thing I notice is you're the first to include skip links! Wasn't a requirement for this, but definitely shows me you know what you're doing 🙌 

I'll do the rest of my checks and provide more feedback

**Comment by thatkidmadrid** [comment]
## TL;DR
- `<figcaption>` tags don't require extra tags to contain text
- watch out for typos
- think of articles as chunks that could stand alone
  - if you have multiple articles, do they make more sense together?

Great accessibility and descriptions overall, but I do have a few things to note and address:
- text inside a `<figcaption>` should not also be wrapped in a `<p>` ([w3 docs on figcaption](https://www.w3schools.com/tags/tag_figcaption.asp))
- careful with typos in your alt attributes (sometimes the screenreader will pronounce things incorrectly which may be confusing to the listener; it wasn't too bad in this case)
- `<article>` tags specify content that should make sense on their own ([w2 docs on articles](https://www.w3schools.com/tags/tag_article.asp)). Although you can argue that the header, main, and footer could make sense on their own for some edge cases, what makes the most sense in my opinion is to have them together in one article. 
  - Wrapping the secondary header that contains the current chapter and nav in an article may not make the most sense. It makes the most sense if it's next to the content of that chapter! I would put it inside the article that contains the div you made with role="main", and then also move the footer with the nav into the same article. This will create a more complete article (with a header-main-footer structure that we're used to seeing)

```
<body>
  <ul class="skip-links">...</ul> <!-- Stoked to see this! -->
  <header role="banner"></header>
  <article>
    <header>...(this contains the main navigation with the role="navigation")</header>
    <div role="main">...</div>
    <footer>(this is the navigation footer)</footer>
  </article>
  <footer role="contentinfo">...</footer>
</body>
```
You've more than demonstrated to me that you understand the accessibility side of the activity so, unless you want to, it's not necessary to make these structural changes, I'll add the grade to LTW soon.

**Comment by KarenMendozaGarza** [comment]
Oh yes, that makes so much more sense! I couldn't figure out why the css had article>footer, so i created one for it to work, but what you're saying makes perfect sense. I have made all of the changes. Thanks for looking over it so quick!

And btw, I just wanted to say that I think you're doing a great job. It's not lost on any of us that teaching remotely is so much more complicated. I hope you know that we all really appreciate the effort you're putting into our learning. Thanks for putting up with my million questions every class. 

See you in two weeks! :)
